W-20825082-formats-functions-systProps-211-dm (#547)
mulesoft/docs-dataweave: modules/ROOT/pages/dw-math-functions-decimaldivide.adoc

== decimalDivide&#40;dividend: Number, divisor: Number, ctx: OperationContext = DECIMAL_128_CONTEXT&#41;: Number

Performs number division with rounding specified by the operation context. If precision is set to 0 (unlimited precision)

> and the result has an infinite decimal expansion it will error.

Reviewer: ```suggestion
and the result has an infinite decimal expansion, the function returns an error.
```
